### PR TITLE
DDF-3265 Upgrade Artemis to 2.2.0

### PR DESCRIPTION
**Pull request by roelens8**
#### What does this PR do?
Upgrades ActiveMQ Artemis from version 2.1.0 to 2.2.0.
#### Who is reviewing it? 
@harrison-tarr 
@ryeats 
@jrnorth 
@derekwilhelm 
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@jaymcnallie
@kcwire 
#### How should this be tested? (List steps with links to updated documentation)
Run a full build with itests passing. Verify that failover still works.
#### What are the relevant tickets?
[DDF-3265](https://codice.atlassian.net/browse/DDF-3265)
#### Checklist:
- [N/A] Documentation Updated
- [] Update / Add Unit Tests
- [] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/ddf-common/src/main/resources/etc/artemis.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
         <network-check-URL-list>http://www.apache.org</network-check-URL-list> -->
 
         <network-check-period>10000</network-check-period>
-        <network-check-timeout>255</network-check-timeout>
+        <network-check-timeout>1000</network-check-timeout>
```
Comment (ryeats) [inline distribution/ddf-common/src/main/resources/etc/artemis.xml]: revert this change linux doesn't support a timeout this large for the network check, can you also add a comment here that 255 is the max support by linux distro's for me.

Comment (roelens8) [reply distribution/ddf-common/src/main/resources/etc/artemis.xml]: fixed
